java/blade/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.example</groupId>
+    <artifactId>benchmark</artifactId>
+    <version>0.0.1</version>
+
+    <properties>
+        <blade-mvc.version>2.0.15.RELEASE</blade-mvc.version>
```

Comment:
rather use `[2.0,2.1)` to keep in **2.0**

Reply:
I noticed changes between 2.0.12 and 2.0.15. Not sure how they version this. I think we can keep fixed version for now.

Reply:
In fact I use this king of interval for all `java` **frameworks**.
For example https://github.com/the-benchmarker/web-frameworks/blob/df8af2046ff4726821784d84e0dccac7fdc61560/java/jooby/pom.xml#L18 says `jooby` to stay in **2.8**. This keep the parity in version displayed in results
I prefer to rely to http://maven.apache.org/enforcer/enforcer-rules/versionRanges.html.

If there is changes (note that it sould not have any **breaking**), I think we rather fix them before merge

Reply:
Ok.